### ISSUE TITLE
Error exporting database with automatic backup

### ISSUE THREAD

**Issue by alexmm11** [orientechnologies/orientdb#6767]
## OrientDB Version, operating system, or hardware.
- v2.2.10

## Operating System
- [x] Linux - Ubuntu 14.04 Digital Ocean virtual instance
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

## Expected behavior and actual behavior
At time of execute the automatic backup, the log shows this errors related to an spatial index:

[orient-server.log.0.txt](https://github.com/orientechnologies/orientdb/files/504646/orient-server.log.0.txt)

And this is my config:

```javascript
{
  "enabled": true,
  "mode": "EXPORT",
  "exportOptions": "",
  "delay": "4h",
  "firstTime": "13:00:00",
  "targetDirectory": "/var/orientdbBackups",
  "targetFileName": "${DBNAME}-${DATE:yyyyMMddHHmmss}.zip",
  "dbInclude": "BH",
  "compressionLevel": 9,
  "bufferSize": 1048576
}
```

The backup file is created, but i get those errors :confused:

**Comment by maggiolo00** [comment]
hi @alexmm11 

do you have manual indexes?

**Comment by alexmm11** [comment]
No, all my indexes are automatic, all were created with the schema.

**Comment by alexmm11** [comment]
When the export is manual (export database etc etc) i dont get any error

**Comment by maggiolo00** [comment]
ok @alexmm11 

i will try to replicate it.
Thanks

**Comment by maggiolo00** [comment]
hi @alexmm11 
Fixed. It will be available in 2.2.12
Thanks

**Comment by alexmm11** [comment]
Thanks man!
